ocs_ci/ocs/resources/pod.py

```diff
@@ -176,6 +176,37 @@ def exec_cmd_on_pod(
             rsh_cmd, out_yaml_format, secrets=secrets, timeout=timeout, **kwargs
         )
 
+    def exec_cmd_on_container(
+        self,
+        container_name,
+        command,
+        out_yaml_format=False,
+        secrets=None,
+        timeout=600,
+        **kwargs,
+    ):
+        """
+        Execute a command on a container (e.g. oc rsh)
+
+        Args:
+            container_name (str): The container name
+            command (str): The command to execute on the given pod
+            out_yaml_format (bool): whether to return yaml loaded python
+                object OR to return raw output
+
+            secrets (list): A list of secrets to be masked with asterisks
+                This kwarg is popped in order to not interfere with
+                subprocess.run(``**kwargs``)
+            timeout (int): timeout for the exec_oc_cmd, defaults to 600 seconds
+
+        Returns:
+            Munch Obj: This object represents a returned yaml file
+        """
+        cmd = f"exec {self.name} -c {container_name} {command}"
```

Comment:
Why do we want to exec into pod instead of rsh?

Reply:
we need to run the cmd on specific container in pod

Reply:
done